docs/content/Cube-Cloud/Configuration/Connecting-to-Databases.md

```diff
@@ -33,6 +33,31 @@ configuration in your `cube.js` file according to your
 [multitenancy setup](/multitenancy-setup). Next, configure the corresponding
 environment variables on the **Settings - Env Vars page**.
 
+## Connecting via SSL
+
+To set up SSL connections between Cube Cloud and your database server, go to
```

Comment:
Make this section a Custom SSL certificate setup section. Mention normal SSL checkbox